packages/webpack/src/plugin.js

```diff
@@ -111,6 +111,20 @@ class LavaMoatPlugin {
     // compiler.options.optimization.moduleIds = "hashed";
     // compiler.options.optimization.chunkIds = "named";
 
+    // sadly regular webpack compilation doesn't allow for synchronous resolver.
```

Comment:
I'm not a huge fan of leaving commented-out code around.  should this be a draft PR, since it's not targeting `main`?